File: APIQuanLyCuaHang/APIQuanLyCuaHang/ViewModels/DataChart/UserStatisticsData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace APIQuanLyCuaHang.ViewModels.DataChart
{
    public class UserStatisticsData
    {
        public List<UserStatistics>? Users { get; set; }
    }

    public class UserStatistics
    {
        public string? UserType { get; set; }
        public int Active { get; set; }
        public int Inactive { get; set; }
    }
}

File: QuanLyCuaHangUXUI/QuanLyCuaHangProject/src/models/dto/dataCharts/UserStatisticsData.js
export default class UserStatisticsData {
  constructor(users = null) {
    this.users = users // users là một mảng các đối tượng UserStatistics
  }
}

export class UserStatistics {
  constructor(userType = null, active = 0, inactive = 0) {
    this.userType = userType
    this.active = active
    this.inactive = inactive
  }
}
